test/PhlyRestfullyTest/CollectionIntegrationTest.php

```diff
@@ -220,7 +221,7 @@ public function testCollectionLinksIncludeFullQueryString()
 
     public function getServiceManager()
     {
-        $controllers = new ControllerManager();
+        $controllers = new ControllerManager(new Config());
```

Comment:
Should likely pass a mock ContainerInterface instance or a ServiceManager instance, as the initial argument starting in zend-servicemanager v3 is the parent container. I'll make that change on merge.